### PR TITLE
Add GARDEN pages

### LINKED ISSUE #1210: Garden subsite - What is Project Garden?
Create a new site view, at the URL childrenhelpingscience.com/GARDEN/about

This is a subsite of the Project Garden pages.  It should have the same Project Garden nav bar at the top as the other pages.  

This one is formatted like the lookit.mit.edu/FAQ page, with some front matter. After the FAQ header, use the same format and expand/hide view as the primary FAQ page. 

Full text below: 

# What is Project Garden?

Project GARDEN is a research project by Children Helping Science and funded by the National Science Foundation. We have two goals: (1) to make it easier for families to participate in research and (2) to better understand how children develop over time in lots of different ways! 

Project GARDEN is a project aimed at understanding child development from many different angles: learning how to relate to their family and peers, how to problem solve, how to speak their family’s languages, and much more!

Each time you participate, your child will have the opportunity to follow a fun story involving a magical library and a friendly fennec fox named Fen!

# Frequently Asked Questions

## What is Project GARDEN?

Project GARDEN is a research project by Children Helping Science and funded by the National Science Foundation. We have two goals: (1) to make it easier for families to participate in research and (2) to better understand how children develop over time in lots of different ways! 

## Why is Project GARDEN important?

Project GARDEN is a project aimed at understanding child development from many different angles! Usually, a research project will only look at one or two specific areas of development, like specific math skills or specific vocabulary skills. In Project GARDEN, we look at how children grow in many different ways at once - learning how to relate to their family and peers, how to problem solve, how to speak their family’s languages, and much more! 

Every time you complete a Project GARDEN study, you will be helping science get a more complete picture of how children grow and develop!

## Is Project GARDEN for me?

Project GARDEN is for all families with 3- to 6-year-old children who are interested in development and education research and would like to help science by participating in lots of small studies! Our goal is for this experience to be engaging and fun for children and interesting and rewarding for caregivers. If you are interested in seeing how your child plays the research games or understanding more about how children learn and develop, you might find Project GARDEN a great experience for you and your family!

## How is Project GARDEN different from other studies on Children Helping Science’s website?

There are many exciting studies on Children Helping Science from around the world, and you can always participate in any studies that interest your family. Project GARDEN is a more specific set of studies on Children Helping Science. Keep reading to find out more!

## How does participating in Project GARDEN work?

Project GARDEN works by families participating in many short studies that, combined together, help us understand children’s development from many different angles. Each time you participate, your child will have the opportunity to follow a fun story involving a magical library and a friendly animated fox,  play research games designed to be fun for children, and receive a gift card. There will be at least 8 studies posted in the upcoming year, and most studies will take 10- 15 minutes to complete. You can choose whether to participate in each study, but the more studies that families complete, the better we can understand how children develop! If you complete all of the first set of 8 studies, you will receive an extra gift card as a thank you. You will then also have the opportunity to participate in more studies in the future!

## Why is it important to complete all of the studies?

The goal of Project GARDEN is to better understand the many different ways children develop using many quick, small studies instead of asking children to sit down for one really long study. Because of this, it’s very important to us for families to try to complete each of the first eight studies so we can look at these studies together! For example, you might do one study on how children think about different people’s perspectives and then another study on what kinds of words they know. Using both studies, we can see how what words they know might relate to how they think about different people’s perspectives. In another example, children might also complete the same study twice-once when they’re young and once when they're a bit older- to see what changed over time!

## Are you curious about participating but want to talk to a researcher first?

Email us at [project garden email] or sign up for a brief talk on [Microsoft Teams](https://calendly.com/utdthinklab/project-garden-question-session) with one of our Project GARDEN researchers and we’ll be happy to answer any questions! Sign-up slots are limited, but we will always respond to your emails within 7 business days. 

## Want to see what studies are like before you sign up?

There are other studies posted on Children Helping Science that are not part of Project GARDEN. You can visit the Children Helping Science [Studies Page](http://childrenhelpingscience.com/studies) to check out other studies to preview or try out another study before you decide if you’d like to participate in Project GARDEN!

### LINKED ISSUE #1211: Garden Subsite - Main page
Create a new site view, at the URL childrenhelpingscience.com/GARDEN/

This is the 'main' page for the new Project Garden pages. This and all 4 pages should have a nav bar at the top which is visible across these pages, as follows: 

Project Garden - childrenhelpingscience.com/GARDEN/
About - childrenhelpingscience.com/GARDEN/about
How to Join - childrenhelpingscience.com/GARDEN/participate
Start Now - childrenhelpingscience.com/studies/garden
The Scientists - childrenhelpingscience.com/GARDEN/scientists

(Note that the Start Now link goes to the existing lab page for project garden! See here: http://lookit.mit.edu/studies/garden)

Full content of the main page starts here: 

********

[Navigation bar: Project Garden | About | How to Join | Start Now | The Scientists]

[Project Garden logo: see Canva]

Welcome to Project GARDEN!

We are looking for families with children who are 3-, 4-, 5-, or 6 years-old to help us in **G**rowing and **A**dvancing **R**esearch in **D**evelopment and **E**ducatio**N** (**GARDEN**).  We have lots of studies that are designed to be engaging and often fun research games for children that are all about learning how children grow and develop in lots of different ways. Project GARDEN works by families participating in many short studies that, combined together, help us understand children’s development from many different angles.
 
Over the next 4 months, we will have at least 8 studies (10-15 minutes each) available in Project GARDEN, and families that do all 8 of our first studies will receive a $10 bonus. There will be additional study opportunities in the future! You can choose whether to participate in each study, but the more studies that families complete, the better we can understand how children develop!

[Fox-on-bridge image: See Canva]

Children who participate will experience a fun storybook where they will meet Fen, a fennec fox who is the new librarian for a magical GARDEN library. But Fen needs help finding it! As children finish research games, they will help Fen fill out a map and travel through many interesting locations on their way to the GARDEN library.

Click the links below to learn more about Project GARDEN!

[What is Project GARDEN?](childrenhelpingscience.com/GARDEN/about)

[New to Project GARDEN? Visit our How to Participate page!](childrenhelpingscience.com/GARDEN/participate)

[Project GARDEN Studies](childrenhelpingscience.com/studies/garden)

[Meet the scientists!](childrenhelpingscience.com/GARDEN/scientists)

### LINKED ISSUE #1212: Garden subsite - How to participate?
This is a new site view, part of the Project Garden subsite, appearing at childrenhelpingscience.com/garden/participate

It should have the same nav bar as all subsites.  

Text begins below!

*********************

# How to Participate

## Is Project GARDEN for me?

Project GARDEN is for all families with 3- to 6-year-old children who:

* Can be supervised by a legal caregiver (parent or other legal guardian) as they participate in each study
* Can use and understand English
* Have access to a computer or laptop that has:
** A webcam and a microphone
** Internet access
* Would like to receive a $5 Amazon.com (U.S.-based) gift card each time they participate in one of our 8 studies.

## Getting started in Project GARDEN

 To start participating in Project GARDEN, you will follow an easy three-step process:

### 1. Create  an account on Children Helping Science

* Click on the “Sign-up” button here [link to Lookit Sign-up page], or at the top right of this website
* Follow the instructions to sign up for an account, including reading and agreeing to the Children Helping Science Privacy Statement
* Note: You will log in to your Children Helping Science account each time you and your child participate in a Project GARDEN study. 

### 2. Visit the Project GARDEN [Studies Page](https://lookit.mit.edu/studies/garden/)

* Click on the first study to get started! 
* You can return to this page at any time to participate in more studies, or, after you’ve completed enough studies, re-watch some Storybook videos with Fen, our friendly fennec fox!
* At the start, only three studies will be available, but more studies will become available each month. For the first part of Project GARDEN, there will be 7 studies for your child and 1 study for you, for a total of 8 studies. More studies will be released over time!

### 3. Click on the first Project GARDEN study to start participating!

* At the start of each study, we will provide a consent form that will tell you all about the study.
* After you read the consent form, you will record a short consent video with you and your child where you say you agree to participate in the study.
* Then, you will start the study– each study will tell you exactly what to do to participate!
* Once you finish, we will check your consent video to make sure you and your child are both present and eligible for the study. All eligible families will receive a $5 Amazon.com gift card (U.S.-only) within 7 business days after participating in each study.
* Families who complete all 8 studies will receive an extra $10 Amazon.com gift card! In total, families who complete all 8 studies will receive $50 in Amazon.com gift cards (eight $5 gift cards and one extra $10 gift card).
* More studies will be available over time, with more adventures with Fen!

Click [here](http://childrenhelpingscience.com/GARDEN-about) to learn more about Project GARDEN and see our FAQ

### LINKED ISSUE #1213: Garden subsite - meet the scientists
This is a new site view, part of the Project Garden subsite, appearing at childrenhelpingscience.com/garden/scientists

It should have the same nav bar as all subsites.

It should be formatted as the current Lookit meet the scientists page, a series of portraits with blurbs underneath. 

TEXT FINALIZED

*****************

# Meet the Scientists

Project GARDEN would not be possible without the work of many scientists and collaborators! Meet the team working hard to help families contribute to developmental science.

## Leadership Team

Candice Mills is the co-director of the Project GARDEN team. Dr. Mills is a full professor of psychology and the director of the Think Lab at the University of Texas at Dallas.

Mark Sheskin is the co-director of the Project GARDEN Team. Dr. Sheskin is an associate professor of psychology at Minerva University.  

Melissa Kline Struhl is on the Project GARDEN leadership team bringing her expertise using and maintaining Children Helping Science’s technical engine called “Lookit” (the code that enables the studies to work!). Dr. Kline Struhl is the Executive Director of Lookit at the Massachusetts Institute of Technology (MIT). 

Ian Chandler-Campbell is a postdoctoral researcher on the Project GARDEN team working at UT Dallas. Dr. Chandler-Campbell is in charge of developing and supporting the family experience and research infrastructure for Project GARDEN. 

## Grant Team Members

Elizabeth Bonawitz is the David Vitale Associate Professor in Learning Sciences and PI of the Computational Cognitive Development Lab at Harvard University, Graduate School of Education.

Hyowon Gweon is an associate professor at Stanford University in the Department of Psychology.

Julian Jara-Ettinger is an associate professor on term at Yale University in the Department of Psychology.

Marjorie Rhodes is a full professor at New York University and the Director of the Conceptual Development and Social Cognition Lab.

Laura Schulz is a full professor at MIT and the PI of the Early Childhood Cognition Lab.

## Research Partners

Nadia Chernyak is an assistant professor in the Department of Cognitive Sciences at the University of California, Irvine and directs the Development of Social Cognition Lab.

Jenny Wang is an assistant professor in the Department of Psychology at Rutgers University and directs the Cognition and Learning Center.

Moira Dillon is an assistant professor at New York University and director of the Lab for the Developing Mind.

Sabine Doebel is an assistant professor at George Mason University and director of the Developing Minds Lab. 

Michael Frank is the David and Lucile Packard Professor of Human Biology at Stanford University and director of the Language and Cognition Lab. 

## Other Team Members

Elham Doust is the animator and lead artist for the GARDEN Storybook featuring Fen the friendly fox! She is a lecturer at UT Dallas in the School of Arts, Humanities, and Technology.

Rosie Aboody is a postdoctoral fellow in Dr. Bonawitz’s Computational Cognitive Development Lab at Harvard, and Dr. Schulz’s Early Childhood Cognition Lab at MIT.

Bria Long is a postdoctoral fellow at Stanford University and a member of the Language & Cognition Lab.

Michelle Wong is a graduate student in Dr. Bonawitz’s Computational Cognitive Lab at Harvard University, Graduate School of Education.

Deisy Martinez is a lab manager in Dr. Moira Dillon’s Lab for the Developing Mind at New York University.

### PR DESCRIPTION
**Pull request by becky-gilbert**
Fixes #1210 
Fixes #1211 
Fixes #1212 
Fixes #1213 

This PR adds the GARDEN pages.
- childrenhelpingscience.com/garden (main/home page)
- childrenhelpingscience.com/garden/about (what is project GARDEN?)
- childrenhelpingscience.com/garden/participate (how to participate)
- childrenhelpingscience.com/garden/scientists (meet the scientists)

Notes:
- All pages have the following navigation tabs above the main content:
  - "Project GARDEN" -> /garden
  - "About" -> /garden/about
  - "How to Join" -> /garden/participate
  - "Start Now" -> /studies/garden (existing lab page)
  - "The Scientists" -> /garden/scientists
- All pages are mobile-friendly
- While working on the GARDEN Scientists page, I put the boilerplate HTML for staff/scientist/student profiles into a template tag and took the opportunity to refactor the main CHS Scientists page - see [next comment](https://github.com/lookit/lookit-api/pull/1221#issuecomment-1550396539).

## Main page

Desktop/laptop:

Screenshot 2023-05-16 at 2 07 42 PM
Screenshot 2023-05-16 at 2 07 52 PM

Mobile:

Screenshot 2023-05-16 at 2 10 30 PM

## About

Desktop/laptop:

Screenshot 2023-05-16 at 2 08 02 PM
Screenshot 2023-05-16 at 2 08 11 PM
Screenshot 2023-05-16 at 2 08 17 PM

Mobile:

Screenshot 2023-05-16 at 2 10 59 PM

## Participate

Desktop/laptop:

Screenshot 2023-05-16 at 2 08 28 PM
Screenshot 2023-05-16 at 2 08 47 PM

Mobile:

Screenshot 2023-05-16 at 2 11 24 PM
Screenshot 2023-05-16 at 2 11 31 PM

## Scientists

Desktop/laptop:

Screenshot 2023-05-16 at 2 09 00 PM
Screenshot 2023-05-16 at 2 09 10 PM
Screenshot 2023-05-16 at 2 09 18 PM
Screenshot 2023-05-16 at 2 09 29 PM
Screenshot 2023-05-16 at 2 09 34 PM

Mobile:

Screenshot 2023-05-16 at 2 10 08 PM

### CODE REVIEW

**Comment by becky-gilbert** [comment]
Because I moved the staff profile HTML into a template tag for the GARDEN scientists page, I refactored the main CHS scientists page using this same template tag. The page looks basically the same on desktop/laptop but is now more mobile-friendly.

I also subbed in the new photos for people who are both pages (as requested [here](https://github.com/lookit/lookit-api/pull/1208#pullrequestreview-1428857971)).

Here are the specific changes:
- Candice Mills' photo replaced with the one used on the GARDEN sites
- Julian Jara-Ettinger's bio now matches the one used on the GARDEN site (NB: I changed this because his position/title differed between the two pages, so I assumed the older CHS one was out of date)
- Mark Sheskin's bio now matches the one used on the GARDEN site (as requested [here](https://github.com/lookit/lookit-api/pull/1208#issuecomment-1544639526))
- Image files are now stored in the static images directory (see #1102) and follow the same naming convention

Note that the "Affiliated Universities and Researchers" section is NOT mobile-friendly (I was trying to avoid scope creep!). But I can make that change too if you want @mekline?

## CHS Scientists

**Desktop/laptop**

Screenshot 2023-05-16 at 2 25 02 PM
Screenshot 2023-05-16 at 2 25 14 PM
Screenshot 2023-05-16 at 2 25 26 PM

**Mobile**

Screenshot 2023-05-16 at 2 26 12 PM
Screenshot 2023-05-16 at 2 26 24 PM
Screenshot 2023-05-16 at 2 26 39 PM
Screenshot 2023-05-16 at 2 26 51 PM
Screenshot 2023-05-16 at 2 27 08 PM

File: web/templates/web/garden/_navigation.html
Comment (becky-gilbert) [inline web/templates/web/garden/_navigation.html:11]: @okaycj do you know if it's possible to pass the link to the GARDEN lab site (`/studies/garden`) into the `nav_link` template tag here? I tried but couldn't figure it out :frowning_face:

**Comment by ianchandlercampbell** [comment]
Ian Feedback:

First of all this all look amazing! I'm quite excited to see this subsite when it goes live. I have three suggestions:

1. **Text Size**: I like the normal text scaling/size on mobile but it looks very small on the desktop site. Could we make that font bigger so it is more readable?

2. **Picture Scaling** I see why the Storybook picture and (on mobile) the Scientist pictures are scaled to fit the width of the text columns, but I wonder if we could reduce the scaling on these so they more readily fit on a single page. I would love the Storybook picture to fit on the first page of the desktop site and for at least one scientist profile to fit on the mobile site (instead of just most of their picture).

3. **Scientists Profile Buffer** I think the buffer between rows on the Scientists page is too narrow. Could more white space be added between the rows so the scientist profile pictures don't look like they're nearly colliding with previous row?

Thank you!

**Comment by ianchandlercampbell** [comment]
@mekline Ok! The default text size is great, so I rescind my first request!

File: web/templatetags/web_extras.py
Comment (okaycj) [inline web/templatetags/web_extras.py:127]: The a-tag element is the same in both cases.  This isn't critical, but it might be better to save the a-tag to a variable and wrap it in a li-tag if list is True.

**Comment by becky-gilbert** [comment]
Thanks for the feedback @ianchandlercampbell! I'm working on this now and will get back to you soon.

File: web/templates/web/garden/_navigation.html
Comment (okaycj) [reply web/templates/web/garden/_navigation.html:11]: You have an example of what I would try down below. 
```
{% url "web:lab-studies-list" lab_slug="garden" as garden_url %}
```

File: web/templates/web/garden/home.html
Comment (okaycj) [inline web/templates/web/garden/home.html]: Why does this need the `base_url`?

File: web/templates/web/garden/scientists.html
Comment (okaycj) [inline web/templates/web/garden/scientists.html:25]: If `subheading_classes` is just a list of BS5 classes, then you could create a new class instead.

File: web/templatetags/web_extras.py
Comment (okaycj) [inline web/templatetags/web_extras.py]: I like this!

**Review by okaycj — APPROVED** [review]
This works!  Just a few questions.

File: web/templatetags/web_extras.py
Comment (becky-gilbert) [reply web/templatetags/web_extras.py:127]: Yep that is a better way. Thanks!

File: web/templates/web/garden/scientists.html
Comment (becky-gilbert) [reply web/templates/web/garden/scientists.html:25]: Yes, will do!
BTW I think there are a few other template tags that are just simple lists of classes. I'll open a separate issue/PR to change those too.

File: web/templates/web/garden/_navigation.html
Comment (becky-gilbert) [reply web/templates/web/garden/_navigation.html]: Ah, so I forgot to mention that I did try exactly what you suggest here, with the nav link like this:
```
{% nav_link request garden_url 'Start Now' list=True %}
```
But that gave me this error when trying to load any page that uses the garden navigation: 
> Reverse for '/studies/garden/' not found. '/studies/garden/' is not a valid view function or pattern name.

The reason is that the `url` template tag returns the absolute path, i.e. `/studies/garden`, but the `nav_link` template tag expects the url argument to be in the form of a name for reverse lookup.

I just now realized that instead of trying to get the GARDEN studies page in the right form for reverse lookup, I could just edit the `nav_link` template tag so that it allows the option of receiving an absolute path. Does that seem like a reasonable solution to you?

File: web/templates/web/garden/home.html
Comment (becky-gilbert) [reply web/templates/web/garden/home.html]: It doesn't 🤦‍♀️  Thanks for catching this - fixed!

**Review by mekline — CHANGES_REQUESTED** [review]
This all looks great! in addition to Ian's comments, Mark has made some wording changes currently being discussed on email, I'll relay actual changes once they resolve conversation!

**Comment by becky-gilbert** [comment]
@ianchandlercampbell @mekline 
WRT Ian's comment (2) about reducing the size of the storybook image and trying to fit all of the GARDEN home page content on a desktop/laptop screen - getting all the content on the screen is challenging because of the way the content is stacked. So as it's currently laid out, it goes beyond a single page even when the storybook and GARDEN logo images are much smaller. 

Here's what it looks like with smaller images:

Screenshot 2023-05-18 at 4 32 15 PM

Or if you prefer, I can reduce the length of the page by putting the GARDEN logo and page title next to one another, and/or putting the storybook image to one side of the text.

Screenshot 2023-05-19 at 10 37 44 AM

Can you let me know which of these (or what combination of) options you'd prefer? I am admittedly not a graphic designer 😅 so please feel free to be picky about relative image sizing, spacing between elements etc.

**Comment by ianchandlercampbell** [comment]
@becky-gilbert These are really good ideas, which have given me some ideas on iterations. What if we started the title element with "Welcome to" and had the Project GARDEN logo to its right? So it would still read "Welcome to Project GARDEN" while incorporating the logo. Also solves a bit of a redundancy issue that was bugging me.

For the Storybook image, what if we sized that to take half of the column width (with the 2nd and 3rd text paragraphs taking the other half). This might result in some of the paragraphs not being on the first page, but I think it would be accessible while looking visually-appealing.

**Comment by becky-gilbert** [comment]
@ianchandlercampbell @mekline
WRT both of your comments about reducing the size of the Scientist images on mobile, here's what it looks like with 2 people per row instead of 1:

Screenshot 2023-05-18 at 4 09 50 PM

And here's what it looks like with 1 person per row but with smaller images:

Screenshot 2023-05-18 at 4 20 18 PM

Can you let me know which layout you prefer?

**Comment by ianchandlercampbell** [comment]
@becky-gilbert I think I like the 1 person per row with reduced image sizes, but interested to hear what others think!

**Comment by becky-gilbert** [comment]
> These are really good ideas, which have given me some ideas on iterations. What if we started the title element with "Welcome to" and had the Project GARDEN logo to its right? So it would still read "Welcome to Project GARDEN" while incorporating the logo. Also solves a bit of a redundancy issue that was bugging me.

> For the Storybook image, what if we sized that to take half of the column width (with the 2nd and 3rd text paragraphs taking the other half). This might result in some of the paragraphs not being on the first page, but I think it would be accessible while looking visually-appealing.

Makes sense! How's this?

Screenshot 2023-05-19 at 11 26 15 AM

The only thing is that this layout causes the image to go beyond the text on wide screens (screenshot above is on a 24 in monitor). As the screen narrows, the text starts to wrap past/around the bottom of the storybook image (see next screenshot).

Screenshot 2023-05-19 at 11 27 19 AM

If that bothers you then I could either (a) make the image a bit smaller on wide screens:

Screenshot 2023-05-19 at 11 10 19 AM

or (b) include the 1st paragraph in the text that wraps around the image:

Screenshot 2023-05-19 at 11 11 47 AM

Screenshot 2023-05-19 at 11 14 44 AM

**Comment by becky-gilbert** [comment]
>3. Scientists Profile Buffer I think the buffer between rows on the Scientists page is too narrow. Could more white space be added between the rows so the scientist profile pictures don't look like they're nearly colliding with previous row?

@ianchandlercampbell how's this?

Screenshot 2023-05-19 at 11 48 30 AM

Screenshot 2023-05-19 at 11 51 21 AM

**Comment by ianchandlercampbell** [comment]
@becky-gilbert These Storybook Background image tweaks look great! I'm not amazing at graphic design either, so interested in what others think, but could we put a boundary so that text that starts in a right or left column stays in that column? I like the 2nd to last option with all text to the left and the Storybook picture to the right, but I don't like the text overflow after the image.

For the Scientists buffer, that looks much better!!

**Comment by becky-gilbert** [comment]
@ianchandlercampbell thanks for the feedback! How's this?

Screenshot 2023-05-19 at 12 02 09 PM

**Comment by mekline** [comment]
(I endorse all of these changes, including the 1-per-row small image Scientists mobile page!)

**Comment by ianchandlercampbell** [comment]
@becky-gilbert I think that's our winner! I love it.

**Comment by becky-gilbert** [comment]
@ianchandlercampbell
Just wanted to recap where we are with this for the sake of clarity.

**GARDEN home page**
- Title is now "Welcome to" \<GARDEN project logo\>
- On desktop/laptop
  - "Welcome to" is to the left of the logo
  - Text is in the left column and storybook image in the right column. The text will stay in the column rather than wrap around the image when the page is resized.
- On mobile
  - "Welcome to" is above the logo
  - Text and storybook image are in a single column. I moved storybook image so that it appears in between the 1st and 2nd paragraphs (I can move it somewhere else if you prefer?).

**GARDEN scientists page**
- On mobile, the profiles still appear as a single column, but the image size is reduced.

**Screenshots**
Screenshot 2023-05-22 at 4 00 19 PM
Screenshot 2023-05-22 at 3 59 43 PM
Screenshot 2023-05-22 at 4 00 54 PM

@mekline did you want me to merge this into `develop` now so that the GARDEN team can review on staging, or wait for further feedback?

**Comment by becky-gilbert** [comment]
@okaycj you already reviewed this PR (thank you!) but I changed the staff profile template tag a bit to simplify it and adjust the image size depending on layout. Before I had the profile div col widths integers being passed in as arguments, but this made things complicated since I wanted to adjust the image size depending on the profile div size. So here's what I came up with:
- just two options for sizing/layout
- use different image sizing for these two options on mobile (full width images when profiles are in rows of 2, reduced width images when profiles are in rows of 1)
Could you please have a look at the staff template tag again and let me know what you think? Thanks!

**Comment by mekline** [comment]
I'm meeting with the team in 6 minutes, let me check on the copy changes they had mentioned!

**Comment by mekline** [comment]
Copy changes!

- "10-15 minutes each" --> "about 15 minutes each"

- "and families that do all 8 of our first studies will receive a $10 bonus" -> "... in Project Garden. Each study will provide a $5 gift card, and families that do all 8 of our first studies will receive a bonus $10 gift card"

- [project garden email] --> project.garden.study@gmail.com

- "brief talk on Microsoft Teams" --> "brief video call"

**Comment by ianchandlercampbell** [comment]
@becky-gilbert Waiting on sign-off, but let me give you the text ahead of time! For the "Getting Started" section, we've revised to a 5 step process, as below. I updated the Subsite Google Doc if that's helpful for formatting, but no changes to formatting, just two new steps and calling it an "easy five-step process" (instead of a 3-step). If you don't hear otherwise from me by 10 AM ET tomorrow, you can go ahead and add this and ship it as-is.

# Getting started in Project GARDEN

 To start participating in Project GARDEN, you will follow an easy five-step process (click on each step to see specific instructions and more information):
## **(1) Create an account on Children Helping Science**

- Click on the “Sign-up” button here [link to Lookit Sign-up page], or at the top right of this website
- Follow the instructions to sign up for an account, including reading and agreeing to the Children Helping Science Privacy Statement
- Note: You will log in to your Children Helping Science account each time you and your child participate in a Project GARDEN study.

## (2) Choose a time that works for you and your child!

- Each module will be for your child (except for the Family Information Survey), so choose a time that works best for your child! Studies take about 15 minutes, but we recommend you have at least 30 minutes available for your very first study and you’re learning how it works.
- If you have more than one child, they will need to participate one at a time. Your Children Helping Science account will ask you which child is participating– make sure to choose the right child!
- We will need to see your child with you when you record your “consent video” for each study  to make sure everyone is there and ready to participate!

## (3) Get a laptop or other computer to log in to your account 

- You will need a laptop or computer with a webcam and microphone (if you have a separate webcam, this should have a microphone on it already!)
- Project GARDEN studies do not work on phones or tablets.

## (4) Visit the Project GARDEN Studies Page [https://lookit.mit.edu/studies/garden/]
- Click on the first study to get started!
- You can return to this page at any time to participate in more studies, or, after you’ve completed enough studies, re-watch some Storybook videos with Fen, our friendly fennec fox!
- At the start, only three studies will be available, but more studies will become available each month. For the first part of Project GARDEN, there will be 7 studies for your child and 1 study for you, for a total of 8 studies. More studies will be released over time!

## (5) Click on the first Project GARDEN study to start participating!
- At the start of each study, we will provide a consent form that will tell you all about the study.
- After you read the consent form, you will record a short consent video with you and your child where you say you agree to participate in the study.
- Then, you will start the study– each study will tell you exactly what to do to participate!
- Once you finish, we will check your consent video to make sure you and your child are both present and eligible for the study. All eligible families will receive a $5 Amazon.com gift card (U.S.-only) within 7 business days after participating in each study.
- Families who complete all 8 studies will receive an extra $10 Amazon.com gift card! In total, families who complete all 8 studies will receive $50 in Amazon.com gift cards (eight $5 gift cards and one extra $10 gift card).
- More studies will be available over time, with more adventures with Fen!

**Comment by becky-gilbert** [comment]
@ianchandlercampbell great, thanks very much! I'll update now with what you have and wait until tomorrow to hear about any further changes 👍

**Comment by becky-gilbert** [comment]
@ianchandlercampbell quick question. I notice this new bit of text at the start of the getting started section:

> ..five-step process (click on each step to see specific instructions and more information):

Does this mean that you want each step to have collapsible instructions (bullet point items) under the numbered heading? And if so, should the numbered items start out collapsed and expand (i.e. instructions become visible) when clicked?

Apologies if this is all explained in the subsite google doc and I missed it. If that's the case then please feel free to send a link here or on Slack (I checked on my google drive to see if I already have access but couldn't find it). Thanks!

**Comment by becky-gilbert** [comment]
@ianchandlercampbell @mekline I will merge this now so that you can see everything on staging. Here's a recap of the last set of changes I made since the minor copy changes from [Melissa's comment above](https://github.com/lookit/lookit-api/pull/1221#issuecomment-1559690902).

- Changed the Getting Started content from 3-step to 5-step process per [Ian's comment above](https://github.com/lookit/lookit-api/pull/1221#issuecomment-1560138552)
- Converted the Getting Started section (on the participate page) and FAQ section (on the about page) from static content into accordions (click on heading to see text appear below)
- Fixed link to the GARDEN studies page in the step 4 heading in the Getting Started section (broke because of the switch to accordion)
- Added the GARDEN home page link onto the CHS home page.

@ianchandlercampbell could you please double check the Getting Started section in particular once this is on staging, just to make sure all of the content changes are correct? (I haven't posted screenshots of all of the sections here). Thanks!

**Getting Started section: desktop/laptop**

Screenshot 2023-05-24 at 10 30 39 AM
Screenshot 2023-05-24 at 10 30 50 AM
Screenshot 2023-05-24 at 10 31 32 AM

**Getting Started section: mobile**

Screenshot 2023-05-24 at 12 14 39 PM

**FAQ section: desktop/laptop**

Screenshot 2023-05-24 at 12 15 13 PM
Screenshot 2023-05-24 at 12 15 26 PM

**FAQ section: mobile**

Screenshot 2023-05-24 at 12 14 55 PM

**GARDEN link on CHS home page**

Screenshot 2023-05-24 at 12 12 26 PM

**Pull request merged**
